TestProjects/AlembicImporter/Assets/AlembicMeshCustomDataMapper.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections;
```

Comment:
This script is a example on how to map the custom vertex attributes to Mesh UV2-3-4 (Solution Ask)